community/pciutils/build

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+mk() {
+    make \
+        PREFIX=/usr \
+        ZLIB=no \
```

Comment:
Any reason for this?

Reply:
i dont know, as i said I nabbed the package from elsewhere, wanted to post it here and get it up to par for the main repo

Reply:
Does it build fine without it? Sounds like it'll link to system zlib if omitted.

Reply:
check the latest and comment. thanks